src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -248,32 +196,24 @@ export class CalcitePickList {
   // --------------------------------------------------------------------------
 
   getIconType(): ICON_TYPES | null {
-    const { multiple } = this;
-    let type = null;
-    if (this.mode === "configuration" && this.dragEnabled) {
-      type = ICON_TYPES.grip;
-    } else if (this.mode === "selection" && multiple) {
+    let type = ICON_TYPES.circle;
+    if (this.multiple) {
       type = ICON_TYPES.square;
-    } else if (this.mode === "selection" && !multiple) {
-      type = ICON_TYPES.circle;
     }
     return type;
   }
 
   render() {
-    const id = this.el.id || this.guid;
     return (
-      <Host id={id}>
-        <section class={CSS.container}>
-          <header>
-            <calcite-filter
-              data={this.dataForFilter}
-              textPlaceholder={TEXT.filterPlaceholder}
-              onCalciteFilterChange={this.handleFilter}
-            />
-          </header>
-          <slot />
-        </section>
+      <Host>
+        <header>
+          <calcite-filter
+            data={this.dataForFilter}
+            textPlaceholder={TEXT.filterPlaceholder}
```

Comment:
nitpick: Does the filter need to be passed some text for the label?

Reply:
Opened an issue to follow up on this.